modules/govuk/manifests/apps/email_alert_api.pp

```diff
@@ -55,15 +63,16 @@
 #   Number of requests per seconds that the notify delivery worker
 #   is allowed to make (currently 50 requests/s / 6 workers = 8 (ish)
 #
-# [*secret_key_base*]
```

Comment:
Did you mean to lose the documentation for the `secret_key_base` field?

Reply:
I did not - thanks for catching. I'm getting sloppy 😟